test/jobs/cache/browser_channels/responses_for_prefix_test.rb

```diff
@@ -109,7 +109,7 @@ def self.test_order
     assert service.temp_file.present?
     result = Brotli.inflate(File.open(service.temp_file.path, "rb").readlines.join("").slice(4..-1))
     result = PublishersPb::ChannelResponseList.decode(result)
-    assert_equal result.channel_responses[0].site_banner_details.web3_url, "/c/123456dfg6"
+    assert_equal URI.parse(result.channel_responses[0].site_banner_details.web3_url).path, "/c/123456dfg6"
```

Comment:
Note the URL returned in this test is the while url, not just the path @jlbyrne

Reply:
I think just had the env var specified 
`details.web3_url = include_web3 ? "#{ENV["CREATORS_HOST"]}/c/#{public_id}" : ""`